src/AbstractOperations/conditional_operations.jl

```diff
@@ -61,7 +61,7 @@ julia> f(i, j, k, grid, c) = i < 2; d = condition_operand(cos, c, f, 10)
 ConditionalOperation at (Center, Center, Center)
 ├── operand: 2×1×1 Field{Center, Center, Center} on RectilinearGrid on CPU
 ├── grid: 2×1×1 RectilinearGrid{Float64, Periodic, Periodic, Bounded} on CPU with 3×3×3 halo
-├── func: cos (generic function with 30 methods)
+├── func: cos (generic function with 26 methods)
```

Comment:
should probably use `showmethods=false` for `prettysummary` to avoid this pain

Reply:
but we can fix it later

Reply:
Yes, not in this PR